torchgeometry/image/laplacian.py

```diff
@@ -0,0 +1,169 @@
+from typing import Tuple
+
+import math
+import torch
+import torch.nn as nn
+from torch.nn.functional import conv2d
+
+
+def laplacian(window_size):
+    r"""One could also use the Laplacian of Gaussian formula
+        to design the filter.
```

Comment:
why not using laplace formula? we should verify that is compliant with opencv or at least see if can be improved the way to create the kernels

Reply:
see answer before. OpenCV has hardcoded for kernel_size of 3 and for the rest I think it uses Sobel filter coefficients (from what I follow). Please, correct me if I'm wrong.

Reply:
ok, let me verify it